standards/qa-process.md

```diff
@@ -47,33 +48,36 @@
 3. PM / Dev defines the priorities for testing and defines the “Definition of Done”.
 4. Points of contact are defined.
 5. Identify known issues to be ignored or are outside of scope.
+6. Post Sprint Meeting is scheduled
 
 ## Access Given (during kickoff meeting)
 1. Desired browsers and devices to be tested on are identified.
-2. Team is assigned to the project repo on Github in order to post issues and coordinate with developers.
+2. Team is assigned to the project repo on Github in order to post issues and coordinate with developers. (This will be Write access)
 3. If applicable, review for each user types abilities and limitations.
 4. Credentials are given to the team.
 5. If QA is billable to the client, such as for T&M projects, PM will add the QA team to the Tick project.
 
 # Testing Process
 ## For QA:
 
-1. QA Testers will create a “Working” issue to work from, copy the Acceptance Test from the “QA Sprint” issue into the first comment, and use that to check off each tested feature.
-2. Discussions should occur constantly within the “Working” issue, tagging people as necessary.
+1. QA will set up a project board for this sprint by copying from the example board on github (link).
```

Comment:
Is there meant to be a link here?  Or will it be added later?

Reply:
Good question, we should probably look into that!